### PR TITLE
move mapping between sa and sp in csv file

### PR DESCRIPTION
**Pull request by prhod**
Fixing https://github.com/CanalTP/osm-transit-extractor/issues/30 
Modifications are : 
- removing `first_stop_area_id` from stop_point.csv
- adding a `stop_area_stop_point.csv`
- cleanup 
- adding a test for extracting stop_area (with addition of a stop_area in fixture)

### CODE REVIEW

File: tests/stops_csv_tests.rs [hunk 1]
```diff
@@ -107,7 +117,19 @@ pub fn osm_fixture_stoppoints_csv() {
...
+
+#[test]
+pub fn osm_fixture_stopareas_stoppoints_csv() {
+    let osm_path = std::env::current_dir()
+        .unwrap()
+        .join("tests/fixtures/osm_fixture.osm.pbf");
+    let mut parsed_pbf = osmpbfreader::OsmPbfReader::new(std::fs::File::open(&osm_path).unwrap());
+    let stop_areas = osm_transit_extractor::get_stop_areas_from_osm(&mut parsed_pbf);
+    let tmp_dir = TempDir::new("osm_transit_extractor").expect("create temp dir");
+    osm_transit_extractor::write_stop_areas_stop_point_to_csv(&stop_areas, &tmp_dir);
```
Comment (woshilapin) [inline tests/stops_csv_tests.rs]: In those 2 tests, there is no assertion. The only assertion is basically "run that and if it doesn't fail, I'm happy". Maybe you could test for the presence of file, or even test for the number of lines in it?

Comment (woshilapin) [reply tests/stops_csv_tests.rs]: You could do it with something like this.

```rust
use std::{
    fs::File,
    io::{BufRead, BufReader},
};

let file_path = tmp_dir
    .path()
    .join("osm-transit-extractor_stop_areas_stop_point.csv");
assert!(file_path.exists());
let file = File::open(file_path).unwrap();
let reader = BufReader::new(file);
assert_eq!(3, reader.lines().count());
```

Comment (prhod) [reply tests/stops_csv_tests.rs]: thanks for the template, I've added the tests for all the csv files ;)

**Comment by woshilapin** [comment]
And as someone great once said

> please don't forget the release :yum: 

:hand: @nlehuby

**Comment by prhod** [comment]
Yep :) Done, but i've not added the binary :stuck_out_tongue:

**Comment by nlehuby** [comment]
:sob: what's the point of a release without binary ?

**Comment by prhod** [comment]
You are able to make it yourself ;)
I've added it, you can DL it now :p

**Pull request merged**